src/app/ui/studentSelector.jsx

```diff
@@ -0,0 +1,72 @@
+"use client";
+import * as React from "react";
+import { useTheme } from "@mui/material/styles";
+import OutlinedInput from "@mui/material/OutlinedInput";
+import InputLabel from "@mui/material/InputLabel";
+import MenuItem from "@mui/material/MenuItem";
+import FormControl from "@mui/material/FormControl";
+import Select from "@mui/material/Select";
+
+const ITEM_HEIGHT = 48;
+const ITEM_PADDING_TOP = 8;
+const MenuProps = {
+  PaperProps: {
+    style: {
+      maxHeight: ITEM_HEIGHT * 4.5 + ITEM_PADDING_TOP,
+      width: 250,
+    },
+  },
+};
+
+const names = ["Oliver Hansen", "Van Henry", "April Tucker", "Ralph Hubbard"];
+
+function getStyles(name, personName, theme) {
+  return {
+    fontWeight:
+      personName.indexOf(name) === -1
+        ? theme.typography.fontWeightRegular
+        : theme.typography.fontWeightMedium,
+  };
+}
+
+export default function MultipleSelect(props) {
+  const theme = useTheme();
+
+  const handleChange = async (event) => {
+    props.setPersonName(event.target.value);
+    props.setPersonID(props.nameID[props.personName]);
+    const uri =
+      "http://localhost:4000/submissions?student_id=" + props.personID;
```

Comment:
We might want to reconsider this url in future cases. It should probably look something like assignment/assignmentId/submission?student_id